Cargo.toml

```diff
@@ -135,6 +135,7 @@ maxminddb = "0.13.0"
 strip-ansi-escapes = "0.1.0"
 colored = "1.9"
 notify = "4.0.14"
+warp = "0.1.20"
```

Comment:
why is this included?

Reply:
It came from master. It's probably shown as new because it was merged after  `notify = "4.0.14"` and not before.